src/hooks/useUniswapCurrencies.js

```diff
@@ -114,19 +118,16 @@ export default function useUniswapCurrencies({
       defaultInputItemInWallet,
       defaultOutputItem,
     };
-  }, [
-    allAssets,
-    defaultInputAddress,
-    defaultInputAsset,
-    isDeposit,
-    isWithdrawal,
-    underlyingPrice,
-  ]);
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Given that the default input / output assets need to only be set once, removing the deps for now (allAssets keep triggering it otherwise). There will be a better way of doing this, but I see it as out of scope for this PR